Makefile

```diff
@@ -3,7 +3,7 @@ BIN_NAME=terraform-provider-instaclustr
 
 
 # for VERSION, don't add prefix "v", e.g., use "1.9.8" instead of "v1.9.8" as it could break circleCI stuff
-VERSION=1.14.4
+VERSION=1.14.5
```

Comment:
You'll be competing with Yu for this version number. It may need to be changed depending on who merges first